libraries/core/src/main/kotlin/com/bumble/appyx/core/navigation/NavModelExt.kt

```diff
@@ -0,0 +1,24 @@
+package com.bumble.appyx.core.navigation
+
+import com.bumble.appyx.core.withPrevious
+import kotlinx.coroutines.flow.map
+
+fun <NavTarget, State> NavModel<NavTarget, State>.removedElementKeys() =
+    this
+        .elements
+        .withPrevious()
+        .map { values ->
+            val previousKeys = values.previous?.map { it.key }
```

Comment:
`?.map { }.orEmpty()` instead of null check.

Reply:
simplified it but in a different way